assets/components/SelectInput.jsx

```diff
@@ -26,7 +27,7 @@ const SelectInput = ({
                         <option value="">{defaultOption}</option>
                     }
                     {options.map((option) => {
-                        return <option key={option.value} value={option.value}>{option.text}</option>;
+                        return <option key={option.value} value={option.value}>{gettext(option.text)}</option>;
```

Comment:
I think here the text should be translated already

Reply:
Removed, but once I removed it I saw the dropdown became untranslated, will check once more after the merge